src/components/PathwayExercise/ContentEdit/index.js

```diff
@@ -0,0 +1,450 @@
+    //   console.log("temp before", temp);
+    //   temp.splice(index, 0, AddHeader);
+    //   console.log("temp after", temp);
+    //   setCourse(temp);
+    // }
+  };
+
+  const putApiAssessmentCall = () => {
+    //delete the component which value is empty string or empty list
+    const tempCourse = [];
+    for (var i in course) {
+      if (course[i].value.length !== 0) {
+        tempCourse.push(course[i]);
+      }
+    }
+    const stringifiedCourse = JSON.stringify(tempCourse, null, 0);
+    console.log(id, stringifiedCourse, "cc");
+    axios({
+      method: METHODS.PUT,
+      url: `${process.env.REACT_APP_MERAKI_URL}/assessment/${id}`,
+      headers: {
+        "version-code": versionCode,
+        accept: "application/json",
+        Authorization: user.data?.token || "",
+      },
+      data: {
+        content: stringifiedCourse,
+      },
+    }).then((res) => {
+      history.push(
+        interpolatePath(PATHS.PATHWAY_COURSE_CONTENT, {
+          courseId: params.courseId,
+          exerciseId: params.exerciseId,
+          pathwayId: params.pathwayId,
+        })
+      );
+      console.log(res, "res");
+    });
+  };
+
+  useEffect(() => {
+    console.log("Punnu");
+    axios({
+      method: METHODS.GET,
+      url: `${process.env.REACT_APP_MERAKI_URL}/courses/${courseId}/exercises`,
+      headers: {
+        "version-code": versionCode,
+        accept: "application/json",
+        Authorization: user.data?.token || "",
+      },
+    })
+      .then((res) => {
+        console.log("res", res);
+        const course_type = res.data.course.exercises[exerciseId].content_type;
+        setCourseType(course_type);
+        if (course_type === "assessment") {
+          if (
+            res.data.course.exercises[exerciseId].content[0].component !==
+            "questionExpression"
+          ) {
+            res.data.course.exercises[exerciseId].content.splice(0, 0, {
+              component: "questionExpression",
+              type: "python",
+              title: "",
+              value: "",
+            });
+          }
+          if (
+            res.data.course.exercises[exerciseId].content[1].component !==
+            "questionCode"
+          ) {
+            res.data.course.exercises[exerciseId].content.splice(1, 0, {
+              component: "questionCode",
+              type: "python",
+              title: "",
+              value: "",
+            });
+          }
+        }
+        setId(res.data.course.exercises[exerciseId].id);
+        setCourse(res.data.course.exercises[exerciseId].content);
+        const Date = res.data.course.exercises[exerciseId].updated_at;
+        const date = Date.split("T")[0].replace(
+          /(\d{4})-(\d{1,2})-(\d{1,2})/,
+          function (match, y, m, d) {
+            return d + "/" + m + "/" + y;
+          }
+        );
+        var longDateStr = moment(date, "D/ M/Y").format("dddd D MMMM Y");
```

Comment:
Can we change `16/11/2022` to `Wednesday 16 November 2022` using this [function](https://github.com/navgurukul/bhanwari-devi/blob/Feat/contentEditor/src/common/date.js#L235-L237) 
I replaced the code `Date.split` with this function by calling it, but  unable to do the same for `moment(date, "D/ M/Y").format("dddd D MMMM Y");`

Reply:
Sure, but you would call it on `updated_at`, not `date`. So, it'd be `formatInUtc(updated_at, "EEEE dd MMMM yyyy")` (See: https://date-fns.org/v2.0.0-alpha.25/docs/format for the list of format tokens for `date-fns`, which can differ from those from `moment`). However, is there a reason to format it in UTC instead of the user's local time? For example, if it were last modified at 1 AM UTC on 11/17, would we want it to read that it was last modified on 11/16 for me since I'm currently 5 hours behind (11/16 at 8pm EST). If so, you can use `format` instead.

Reply:
There isn't a reason in the course editor but in create class form as backend wants it.

Reply:
`formatInUtc(updated_at, "EEEE dd MMMM yyyy")` here `EEEE` stands for day?